platform/location.hpp

```diff
@@ -28,6 +30,7 @@ namespace location
     EAndroidNative,
     EGoogle,
     ETizen, // Deprecated but left here for backward compatibility.
+    EGeoClue2,
```

Comment:
Looks like it's not serialized in settings, but I wouldn't change the existing order anyway.

Reply:
So can we keep it?